source/github-helpers/get-pr-info.ts

```diff
@@ -1,24 +1,59 @@
+import * as pageDetect from 'github-url-detection';
+
 import * as api from './api';
 import {getConversationNumber} from '.';
 
 type PullRequestInfo = {
-	// TODO: Probably can be used for #3863 and #4679
-	baseRefOid: string;
-
-	// https://docs.github.com/en/graphql/reference/enums#mergeablestate
-	mergeable: 'CONFLICTING' | 'MERGEABLE' | 'UNKNOWN';
-	viewerCanEditFiles: boolean;
+	prInfo: {
+		// TODO: Use this for `restore-file` when GHE supports `compare`
+		baseRefOid: string;
+		// https://docs.github.com/en/graphql/reference/enums#mergeablestate
+		mergeable: 'CONFLICTING' | 'MERGEABLE' | 'UNKNOWN';
+		viewerCanEditFiles: boolean;
+	};
+	comparison: {
+		status: 'BEHIND' | 'DIVERGED' | 'AHEAD' | 'IDENTICAL';
+	};
 };
```

Comment:
You can merge these into flat object, no need to have prInfo and comparison keys. While we're at it it can be simplified further, like setting `diverged: true` instead of matching strings outside

Reply:
> While we're at it it can be simplified further, like setting `diverged: true` instead of matching strings outside

Can you explain this one more time I am not sure I understood what you said